### PR TITLE
Add doni

### PR DESCRIPTION
**Pull request by doniwirawan**

### CODE REVIEW

File: public/data/doniwirawan.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        "name": "Follow me on Instagram",
+        "url": "https://instagram.com/doni_wirawans",
+        "icon": "instagram"
+      },
+      {
+        "name": "My Personal Website",
+        "url": "https://www.doniwirawan.studio",
+        "icon": "website"
+      }
+
```
Comment (rohan-kulkarni-25) [inline public/data/doniwirawan.json:26]: - Please remove this space

File: public/data/doniwirawan.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    ]
+  }
+
```
Comment (rohan-kulkarni-25) [inline public/data/doniwirawan.json:29]: - Please delete the last line ( Empty Space )

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]

**Review by aayush89890 — APPROVED** [review]
Hi @doniwirawan Welcome to the community.

**Comment by doniwirawan** [comment]
> Hi @doniwirawan Welcome to the community.

thanks brother

File: public/data/doniwirawan.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        "name": "Follow me on Instagram",
+        "url": "https://instagram.com/doni_wirawans",
+        "icon": "instagram"
+      },
+      {
+        "name": "My Personal Website",
+        "url": "https://www.doniwirawan.studio",
+        "icon": "website"
+      }
+
```
Comment (Panquesito7) [inline public/data/doniwirawan.json:26]:
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/doniwirawan.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        "url": "https://instagram.com/doni_wirawans",
+        "icon": "instagram"
+      },
+      {
+        "name": "My Personal Website",
+        "url": "https://www.doniwirawan.studio",
+        "icon": "website"
+      }
+    ]
+  }
```
Comment (Panquesito7) [inline public/data/doniwirawan.json:27]:
```suggestion
}
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Comment by doniwirawan** [comment]
> LGTM 🚀 Thank you for your contribution! 😄👍

thanks

**Pull request merged**